src/elm/Api.elm

```diff
@@ -154,7 +154,7 @@ blockInfoDecoder =
 
 getTransactionStatus : Config -> T.TxHash -> (ApiResult TransactionStatusResponse -> msg) -> Cmd msg
 getTransactionStatus cfg txHash msg =
-    getMiddleware cfg ("/v1/transactionStatus/" ++ txHash) transactionStatusResponseDecoder msg
+    getMiddleware { middlewareUrl = "" } ("/v1/transactionStatus/" ++ txHash) transactionStatusResponseDecoder msg
```

Comment:
Why was this change made? As far as I can tell the default config also uses `""` when in production mode.
If this change is necessary, then the `Config` parameter is unused and should be removed 😊

Reply:
this should not have been changed, it was only for debugging.
Nice catch